products/drush.md

```diff
@@ -26,13 +26,20 @@ auto:
         releaseCycle:
           column: "Drush Version"
           regex: '^Drush (?P<value>\d+).*$'
-          regex_exclude: '^Drush 13.*$' # not yet released
         supportedPhpVersions: "PHP Version" # release_table does not support the retrieval of Drupal versions
         eol: "End Of Life"
 
 # Think about updating the regex_exclude above when a new major version is added.
 # Supported PHP / Drupal versions can be found on https://www.drush.org/latest/install/.
 releases:
+-   releaseCycle: "13"
+    releaseDate: 2024-07-18
+    eol: false
+    supportedPhpVersions: "8.3+"
+    supportedDrupalVersions: "11"
+    latest: "13.0.0"
+    latestReleaseDate: 2024-07-18
```

Comment:
should be 08-02?

```suggestion
    latestReleaseDate: 2024-08-02
```

Reply:
Yes, I probably did a copy/paste and forgot to update the dates :man_shrugging: